app/lib/player.rb

```diff
@@ -2,6 +2,7 @@ module Hammeroids
   # Represents player stores player UUID in redis.
   class Player
     attr_accessor :id
+    attr_accessor :name
```

Comment:
you can just do 
```attr_accessor :id, :name```

https://apidock.com/ruby/Module/attr_accessor

Reply:
Nice